### ISSUE TITLE
Roadmap: Moving ZPTX projects onto Skeleton

### ISSUE THREAD

**Issue by zapteryx** [ZPTXDev/Skeleton#71]
### Describe the feature

List of features pending addition before we can move projects onto Skeleton:

- [ ] ~~Handling per-guild data (maybe per-user too?)~~ _Opting for a data manager instead to allow for customizability_
- [x] Handling localized strings
- [x] Embed builder for embed generation with the default format (consistency purposes)
- [ ] ~~Interaction reply handler~~ _Would probably be better for each bot to have their own reply handler_
- [ ] Array Config Item

### Priority

High

### List the benefits of adding such a feature

This will ensure all shared code between Quaver and Warden are covered by Skeleton, meaning less repeated code across repos

**Comment by zapteryx** [comment]
since the goal for Skeleton has since shifted, this roadmap is no longer relevant. projects will still continue to be migrated though
